test/tests/functional/pbs_hook_execjob_prologue.py

```diff
@@ -252,13 +252,14 @@ def test_prologue_hook_does_not_execute_twice_on_pbsdsh(self):
 
         j = Job(PBSROOT_USER, {'Resource_List.select': '2:ncpus=1',
                                'Resource_List.place': 'scatter'})
-        j.create_script('pbsdsh  hostname')
+        j.create_script('pbsdsh  hostname\nsleep 10\n')
         jid = self.server.submit(j)
         attribs = self.server.status(JOB, id=jid)
         self.server.expect(JOB, 'queue', op=UNSET, id=jid)
         host, opath = attribs[0]['Output_Path'].split(':', 1)
-        ret = self.du.cat(hostname=self.mom.shortname, filename=opath,
-                          sudo=True)
+        ret = self.du.cat(hostname=self.mom.shortname, filename=opath)
```

Comment:
should not the hostname be "host" that you are reading from qstat output above?

Reply:
It is one and the same thing. Here we just mean to get the location of the o/p file which is fixed. We do not mean to test the host obtained from the o/p path.

Reply:
I think on systems like CRAY or HPE ICE where mom is not on the same machine I think it makes more sense to use the host from the qstat -f output.